notebooks/tutorials/beginner_workflow/footprint_workflow.ipynb

```diff
@@ -25,7 +25,7 @@
     "import os\n",
     "import matplotlib.pyplot as plt\n",
     "from openghg.localclient import get_obs_surface\n",
-    "from openghg.modules import Emissions, FOOTPRINTS, ObsSurface\n",
+    "from openghg.modules import Emissions, Footprints, ObsSurface\n",
     "from openghg.processing import footprints_data_merge, search\n",
```

Comment:
Similiarly so these functions need to be imported from somewhere else now?

Reply:
I'll change these as well.